app/src/main/java/com/kamron/pogoiv/ScreenGrabber.java

```diff
@@ -62,35 +61,6 @@ public static ScreenGrabber getInstance() {
         return instance;
     }
 
-    @NonNull private static Rect getBitmapBounds(Bitmap bmp) {
-        return new Rect(0, 0, bmp.getWidth(), bmp.getHeight());
-    }
-
-    static @Nullable @ColorInt int[] getPixelsSafe(Bitmap bmp, Point[] points) {
```

Comment:
Nitpick, I don't like moving functions inside a file as part of a commit that's doing something else. Makes it hard to review because all the lines show up as "new".